src/utils/assert.ts

```diff
@@ -13,6 +16,204 @@ const assert = {
       assert.isETHAddressHex(variableName, address);
     });
   },
+  isAddressNotZero(address: string): void {
```

Comment:
It should take a variableName as `isETHAddressHex` so the message should be "${variableName} is not expected to be 0x000" or something like that.